hermes-console/static/partials/modal/editGroup.html

```diff
@@ -27,7 +27,8 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit group:</small
             <div class="form-group {{groupForm.supportTeam.$valid ? '' : 'has-error'}}">
                 <label for="supportTeam" class="col-md-3 control-label">Support team</label>
                 <div class="col-md-9">
-                    <input class="form-control" required id="supportTeam" name="supportTeam" placeholder="comma delimited names of groups that will have rights to manage this group" ng-model="group.supportTeam"/>
+                    <input class="form-control" required id="supportTeam" name="supportTeam" uib-typeahead="group.name for group in groups($viewValue)" typeahead-min-length="3"
```

Comment:
It would be nice if we performed HTTP requests only when user stopped typing, try something like this:
```js
ng-model-options="{debounce: 700}"
```